XdmfWriter.h

```diff
@@ -199,13 +199,12 @@ class XdmfWriter
 
 		// Open the backend
 		m_backend.open(m_outputPrefix, cellVariableData, vertexVariableData, m_timeStep == 0);
+                backup(m_outputPrefix);
```

Comment:
Is it actually correct to move this out of the if (m_rank == 0) clause?

Reply:
Yes, the backup function has an `if (m_rank == 0)` itself.